backend/pycon/settings/base.py

```diff
@@ -202,3 +202,8 @@
 QUERY_INSPECT_TRACEBACK_ROOTS = [root(".")]
 
 PINPOINT_APPLICATION_ID = env("PINPOINT_APPLICATION_ID", default="")
+
+
+MAILCHIMP_SECRET_KEY = env("MAILCHIMP_SECRET_KEY")
+MAILCHIMP_DC = "us5"
```

Comment:
tomorrow morning I will check what is our DC value, if it is us5 too we could just not add it to settings/base.py and just have it hardcoded in the URL but it is the same, it can stay here too